docs/rsdl/rapid-rsdl-capability-mapping.mdx

```diff
@@ -0,0 +1,221 @@
+---
+id: rsdl-capability-mapping
+title: Capability semantics
+sidebar_label: RSDL API Capability semantics
+---
+
+> DRAFT
+> Initial Draft. Dec 2021
+
+Mapping RSDL capabilities to CSDL annotations
+
+## Delete
+
+```rsdl
+path /skus/{id} {
+    DELETE
+}
+```
+
+```xml
+ <Annotations Target="skus">
+  <Annotation Term="Org.OData.Capabilities.V1.DeleteRestrictions">
+    <Record Type="Org.OData.Capabilities.V1.DeleteRestrictionsType">
+      <PropertyValue Property="Deletable" Bool="true" />
+    </Record>
+  </Annotation>
+</Annotations>
+```
+
+## Update
+
+```rsdl
+path /skus/{id} {
+    PATCH {
+        excluded { id releaseDate }
+    }
+}
+```
+
+```xml
+<Annotations Target="skus">
+  <Annotation Term="Org.OData.Capabilities.V1.UpdateRestrictions">
+    <Record Type="Org.OData.Capabilities.V1.UpdateRestrictionsType">
+      <PropertyValue Property="Updatable" Bool="true" />
```

Comment:
```suggestion
```
"true" is the default

Reply:
I am ok to not generate it but would suggest to keep it here to make it explicit. 
If you think this is too distracting I might change it to a comment 
```xml
<!-- by default <PropertyValue Property="Updatable" Bool="true" /> -->
```

Thoughts?

Reply:
You can leave it in the generated output, but I'd remove it from the documentation as it has no effect.